### ISSUE TITLE
Compile error: Cannot import 'delegate_l1_handler' from 'starkware.starknet.common.syscalls'.

### ISSUE THREAD

**Issue by firmianavan** [OpenZeppelin/cairo-contracts#357]
```
(py3env) # cairo-compile -v
cairo-compile 0.9.0
(py3env) # nile version
0.6.1
(py3env) # python --version
Python 3.7.10
```

Seems not compatible with [cairo v0.9.0](https://github.com/starkware-libs/cairo-lang/releases/tag/v0.9.0)

Will you fix that, or I just use a lower version?

**Comment by eddiexbank** [comment]
just notice OZ is following https://github.com/OpenZeppelin/cairo-contracts/pull/364

**Comment by martriay** [comment]
This has been fixed by #364. Will be out on the next release this week.
